<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8">
    <style>
    body {
      max-width: 900px;
      margin: auto;
      padding: 10px;
      margin-top: 5%;
      align-content: center;
    }
    a:link {color: #8D91B4;}
    a:visited {color: #8D91B4;}
    div {
      text-align: justify;
    }

    ol > li::marker {
      font-weight: bold;
    }

    </style>
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
</head>

<div style="float:left; width:10%; padding-bottom: 40px; padding-right: 20px; margin:5px">
  <img src="./img/p.jpg" style="max-width:100%;" />
</div>

<b>Téo Lemane</b><br />
teo.lemane@genoscope.cns.fr<br /><br />
<a href="https://github.com/tlemane"><img src="./img/logo/git20.png" alt="git" /></a><br />
<a href="https://github.com/tlemane"><img src="./img/logo/twitter.png" alt="twitter" /></a>

<div style="clear:both">
  <p>I am a postdoctoral researcher in the <a href="https://lage.genoscope.cns.fr/">LAGE</a> team at <a href="https://jacob.cea.fr/drf/ifrancoisjacob/Pages/Departements/Genoscope.aspx">Genoscope</a>. My research work focuses on algorithms and data structures for large-scale processing of complex sequencing banks as metagenomes from the <a href="https://fondationtaraocean.org/en/home/">Tara projet</a>. Previously I have developed methods and tools for constructing, streaming and filtering k-mers matrices from large sequencing collections.</p>


  <h3>Publications</h3>
  <ol reversed>
    <li><b>T. Lemane</b>, <i>Indexing and analysis of large sequencing collections using k-mer matrices</i>, PhD Thesis, 2022. [<a href="./pdf/phd.pdf"><b>PDF</b></a>]</li>
    <li><b>T. Lemane</b>, R. Chikhi, P. Peterlongo, <i>kmdiff, large-scale and user-friendly differential k-mer analyses</i>, Bioinformatics, 2022. [<a href="./pdf/LemaneKmdiff2022.pdf"><b>PDF</b></a>]</li>
    <li>Y. Dufresne, <b>T. Lemane</b>, P. Marijon, P. Peterlongo, A. Rahman, M. Kokot, P. Medvedev, S. Deorowicz, R. Chikhi, <i>The K-mer File Format: a standardized and compact disk representation of sets of k-mers</i>, Bioinformatics, 2022. [<a href="./pdf/Dufresne2022.pdf"><b>PDF</b></a>]</li>
    <li><b>T. Lemane</b>, P. Medvedev, R. Chikhi, P. Peterlongo, <i>kmtricks: Efficient and flexible construction of Bloom filters for large sequencing data collections</i>, Bioinformatics Advances, 2022. [<a href="./pdf/Lemane2022.pdf"><b>PDF</b></a>]</li>
    <li>G. Siekaniec, E. Roux, <b>T. Lemane</b>, E. Guédon, J. Nicolas, <i>Identification of isolated or mixed strains from long reads: a challenge met on Streptococcus thermophilus using a MinION sequencer</i>, Microbial Genomics, 2021. [<a href="./pdf/Siekaniec2021.pdf"><b>PDF</b></a>]</li>
  </ol>

  <h3>Software</h3>
  <ul>
    <li><i class="fa fa-github"></i> <a href="https://github.com/tlemane/kmtricks"><b>kmtricks</b></a>:
      kmtricks is a tool suite for counting k-mers, and constructing bloom filters or counted kmer matrices from large and numerous read sets.</li>
    <li><i class="fa fa-github"></i> <a href="https://github.com/tlemane/kmdiff"><b>kmdiff</b></a>:
      kmdiff is a tool allowing differential k-mers analysis on large cohorts.</li>
    <li><i class="fa fa-github"></i> <a href="https://github.com/tlemane/kmindex"><b>kmindex</b></a>:
      kmindex is a tool for querying presence/absence k-mer matrices computed by kmtricks.</li>
  </ul>

  <h3>Talks</h3>
  <ul>
    <li><a href="https://jobim2022.sciencesconf.org/"><b>JOBIM 2022</b></a> kmtricks: creating bloom filters for indexing large sequencing data collection. [<a href="./pdf/JOBIM2022.pdf">slides</a>]</li>
    <li><a href="https://seqbim.cnrs.fr/seqbim-2021/"><b>SeqBIM 2021</b></a> Advances in k-mer matrix construction for analysis of large sequencing collections. [<a href="./pdf/seqBim2021.pdf">slides</a>]</li>
    <li><a href="https://seqbim2020.sciencesconf.org/"><b>SeqBIM 2020</b></a> kmtricks: modular k-mer count matrix and Bloom filter construction for large read collections [<a href="./pdf/kmtricks_seqBim2020.pdf">slides</a>]</li>
    <li><a href="https://dsb-meeting.github.io/DSB2020/"><b>DSB 2020</b></a> HowDeSBT and Simka wedding: what as been done and what we plan to do ? [<a href="./pdf/howde_simka_dsb2020.pdf">slides</a>]</li>
  </ul>

  <h3>Teaching</h3>

  <ul>
    <li><b>2019-2021:</b> Strings algorithms and bioinformatics, MSc1 Software Engineering, ISTIC</li>
  </ul>
  </div>
</html>
