<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> publications | Teo Lemane </title> <meta name="author" content="Teo Lemane"> <meta name="description" content=""> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%E2%9A%9B%EF%B8%8F&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://tlemane.github.io/publications/"> <script src="/assets/js/theme.js?9a0c749ec5240d9cda97bc72359a72c0"></script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> <span class="font-weight-bold">Teo</span> Lemane </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about </a> </li> <li class="nav-item "> <a class="nav-link" href="/news/">news </a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">publications <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/repositories/">repositories </a> </li> <li class="nav-item "> <a class="nav-link" href="/talks/">talks </a> </li> <li class="nav-item "> <a class="nav-link" href="/teaching/">teaching </a> </li> <li class="nav-item"> <button id="search-toggle" title="Search" onclick="openSearchModal()"> <span class="nav-link">ctrl k <i class="ti ti-search"></i></span> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">publications</h1> <p class="post-description"></p> </header> <article> <div class="publications"> <h2 class="bibliography">2024</h2> <ol class="bibliography"> <li> <div class="row"> <div id="lemane_indexing_2024" class="col-sm-12"> <div class="title">Indexing and real-time user-friendly queries in terabyte-sized complex genomic datasets with kmindex and ORA</div> <div class="author"> <em>Téo Lemane</em>, Nolan Lezzoche, Julien Lecubin, Eric Pelletier, <a href="https://www.mio.osupytheas.fr/fr/magali-lescot" rel="external nofollow noopener" target="_blank">Magali Lescot</a>, <a href="http://rayan.chikhi.name" rel="external nofollow noopener" target="_blank">Rayan Chikhi</a>, and <a href="https://people.rennes.inria.fr/Pierre.Peterlongo/" rel="external nofollow noopener" target="_blank">Pierre Peterlongo</a> </div> <div class="periodical"> <em>Nature Computational Science</em>, Feb 2024 </div> <div class="periodical"> Publisher: Nature Publishing Group </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://www.nature.com/articles/s43588-024-00596-6" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Link</a> <a href="https://github.com/tlemane/kmindex.git" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank"><i class="fa-brands fa-github"></i></a> </div> <div class="abstract hidden"> <p>Public sequencing databases contain vast amounts of biological information, yet they are largely underutilized as it is challenging to efficiently search them for any sequence(s) of interest. We present kmindex, an approach that can index thousands of metagenomes and perform sequence searches in a fraction of a second. The index construction is an order of magnitude faster than previous methods, while search times are two orders of magnitude faster. With negligible false positive rates below 0.01%, kmindex outperforms the precision of existing approaches by four orders of magnitude. Here we demonstrate the scalability of kmindex by successfully indexing 1,393 marine seawater metagenome samples from the Tara Oceans project. Additionally, we introduce the publicly accessible web server Ocean Read Atlas, which enables real-time queries on the Tara Oceans dataset.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="willink_genomics_2024" class="col-sm-12"> <div class="title">The genomics and evolution of inter-sexual mimicry and female-limited polymorphisms in damselflies</div> <div class="author"> Beatriz Willink, Kalle Tunström, Sofie Nilén, <a href="http://rayan.chikhi.name" rel="external nofollow noopener" target="_blank">Rayan Chikhi</a>, <em>Téo Lemane</em>, Michihiko Takahashi, Yuma Takahashi, Erik I. Svensson, and Christopher West Wheat </div> <div class="periodical"> <em>Nature Ecology &amp; Evolution</em>, Jan 2024 </div> <div class="periodical"> Publisher: Nature Publishing Group </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://www.nature.com/articles/s41559-023-02243-1" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Link</a> <a href="https://github.com/bwillink/Morph-locus" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank"><i class="fa-brands fa-github"></i></a> </div> <div class="abstract hidden"> <p>Sex-limited morphs can provide profound insights into the evolution and genomic architecture of complex phenotypes. Inter-sexual mimicry is one particular type of sex-limited polymorphism in which a novel morph resembles the opposite sex. While inter-sexual mimics are known in both sexes and a diverse range of animals, their evolutionary origin is poorly understood. Here, we investigated the genomic basis of female-limited morphs and male mimicry in the common bluetail damselfly. Differential gene expression between morphs has been documented in damselflies, but no causal locus has been previously identified. We found that male mimicry originated in an ancestrally sexually dimorphic lineage in association with multiple structural changes, probably driven by transposable element activity. These changes resulted in ~900 kb of novel genomic content that is partly shared by male mimics in a close relative, indicating that male mimicry is a trans-species polymorphism. More recently, a third morph originated following the translocation of part of the male-mimicry sequence into a genomic position ~3.5 mb apart. We provide evidence of balancing selection maintaining male mimicry, in line with previous field population studies. Our results underscore how structural variants affecting a handful of potentially regulatory genes and morph-specific genes can give rise to novel and complex phenotypic polymorphisms.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2023</h2> <ol class="bibliography"><li> <div class="row"> <div id="duitamagonzalez_decom_2023" class="col-sm-12"> <div class="title">decOM: similarity-based microbial source tracking of ancient oral samples using k-mer-based methods</div> <div class="author"> <a href="https://camiladuitama.github.io/gradfolio/" rel="external nofollow noopener" target="_blank">Camila Duitama González</a>, <a href="https://rvicedomini.github.io/" rel="external nofollow noopener" target="_blank">Riccardo Vicedomini</a>, <em>Téo Lemane</em>, Nicolas Rascovan, Hugues Richard, and <a href="http://rayan.chikhi.name" rel="external nofollow noopener" target="_blank">Rayan Chikhi</a> </div> <div class="periodical"> <em>Microbiome</em>, Nov 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://doi.org/10.1186/s40168-023-01670-3" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Link</a> <a href="https://github.com/CamilaDuitama/decOM" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank"><i class="fa-brands fa-github"></i></a> </div> <div class="abstract hidden"> <p>The analysis of ancient oral metagenomes from archaeological human and animal samples is largely confounded by contaminant DNA sequences from modern and environmental sources. Existing methods for Microbial Source Tracking (MST) estimate the proportions of environmental sources, but do not perform well on ancient metagenomes. We developed a novel method called decOM for Microbial Source Tracking and classification of ancient and modern metagenomic samples using k-mer matrices.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2022</h2> <ol class="bibliography"> <li> <div class="row"> <div id="phd" class="col-sm-12"> <div class="title">PhD thesis: Indexing and analysis of large sequencing collections using k-mer matrices</div> <div class="author"> <em>Téo Lemane</em> </div> <div class="periodical"> <em>Université de Rennes 1 &amp; Inria</em>, Dec 2022 </div> <div class="periodical"> </div> <div class="links"> <a href="/assets/pdf/papers/phd.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> </div> </div> </li> <li> <div class="row"> <div id="lemane_kmtricks_2022" class="col-sm-12"> <div class="title">kmtricks: efficient and flexible construction of Bloom filters for large sequencing data collections</div> <div class="author"> <em>Téo Lemane</em>, <a href="https://medvedevgroup.com/principal-investigator/" rel="external nofollow noopener" target="_blank">Paul Medvedev</a>, <a href="http://rayan.chikhi.name" rel="external nofollow noopener" target="_blank">Rayan Chikhi</a>, and <a href="https://people.rennes.inria.fr/Pierre.Peterlongo/" rel="external nofollow noopener" target="_blank">Pierre Peterlongo</a> </div> <div class="periodical"> <em>Bioinformatics Advances</em>, Jan 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://doi.org/10.1093/bioadv/vbac029" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Link</a> <a href="/assets/pdf/papers/Lemane2022.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> <a href="https://github.com/tlemane/kmtricks.git" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank"><i class="fa-brands fa-github"></i></a> </div> <div class="abstract hidden"> <p>When indexing large collections of short-read sequencing data, a common operation that has now been implemented in several tools (Sequence Bloom Trees and variants, BIGSI) is to construct a collection of Bloom filters, one per sample. Each Bloom filter is used to represent a set of k-mers which approximates the desired set of all the non-erroneous k-mers present in the sample. However, this approximation is imperfect, especially in the case of metagenomics data. Erroneous but abundant k-mers are wrongly included, and non-erroneous but low-abundant ones are wrongly discarded. We propose kmtricks, a novel approach for generating Bloom filters from terabase-sized collections of sequencing data. Our main contributions are (i) an efficient method for jointly counting k-mers across multiple samples, including a streamlined Bloom filter construction by directly counting, partitioning and sorting hashes instead of k-mers, which is approximately four times faster than state-of-the-art tools; (ii) a novel technique that takes advantage of joint counting to preserve low-abundant k-mers present in several samples, improving the recovery of non-erroneous k-mers. Our experiments highlight that this technique preserves around 8× more k-mers than the usual yet crude filtering of low-abundance k-mers in a large metagenomics dataset.https://github.com/tlemane/kmtricks.Supplementary data are available at Bioinformatics Advances online.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="lemane_kmdiff_2022" class="col-sm-12"> <div class="title">kmdiff, large-scale and user-friendly differential k-mer analyses</div> <div class="author"> <em>Téo Lemane</em>, <a href="http://rayan.chikhi.name" rel="external nofollow noopener" target="_blank">Rayan Chikhi</a>, and <a href="https://people.rennes.inria.fr/Pierre.Peterlongo/" rel="external nofollow noopener" target="_blank">Pierre Peterlongo</a> </div> <div class="periodical"> <em>Bioinformatics</em>, Dec 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://doi.org/10.1093/bioinformatics/btac689" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Link</a> <a href="/assets/pdf/papers/LemaneKmdiff2022.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> <a href="https://github.com/tlemane/kmdiff.git" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank"><i class="fa-brands fa-github"></i></a> </div> <div class="abstract hidden"> <p>Genome wide association studies elucidate links between genotypes and phenotypes. Recent studies point out the interest of conducting such experiments using k-mers as the base signal instead of single-nucleotide polymorphisms. We propose a tool, kmdiff, that performs differential k-mer analyses on large sequencing cohorts in an order of magnitude less time and memory than previously possible.https://github.com/tlemane/kmdiffSupplementary data are available at Bioinformatics online.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="dufresne_k-mer_2022" class="col-sm-12"> <div class="title">The K-mer File Format: a standardized and compact disk representation of sets of k-mers</div> <div class="author"> <a href="https://yoann-dufresne.github.io/" rel="external nofollow noopener" target="_blank">Yoann Dufresne</a>, Teo Lemane, Pierre Marijon, <a href="https://people.rennes.inria.fr/Pierre.Peterlongo/" rel="external nofollow noopener" target="_blank">Pierre Peterlongo</a>, Amatur Rahman, Marek Kokot, <a href="https://medvedevgroup.com/principal-investigator/" rel="external nofollow noopener" target="_blank">Paul Medvedev</a>, Sebastian Deorowicz, and <a href="http://rayan.chikhi.name" rel="external nofollow noopener" target="_blank">Rayan Chikhi</a> </div> <div class="periodical"> <em>Bioinformatics</em>, Sep 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://doi.org/10.1093/bioinformatics/btac528" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Link</a> <a href="/assets/pdf/papers/Dufresne2022.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> <a href="https://github.com/Kmer-File-Format" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank"><i class="fa-brands fa-github"></i></a> </div> <div class="abstract hidden"> <p>Bioinformatics applications increasingly rely on ad hoc disk storage of k-mer sets, e.g. for de Bruijn graphs or alignment indexes. Here, we introduce the K-mer File Format as a general lossless framework for storing and manipulating k-mer sets, realizing space savings of 3–5× compared to other formats, and bringing interoperability across tools.Format specification, C++/Rust API, tools: https://github.com/Kmer-File-Format/.Supplementary data are available at Bioinformatics online.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2021</h2> <ol class="bibliography"><li> <div class="row"> <div id="siekaniec_identification_2021" class="col-sm-12"> <div class="title">Identification of isolated or mixed strains from long reads: a challenge met on Streptococcus thermophilus using a MinION sequencer</div> <div class="author"> Grégoire Siekaniec, Emeline Roux, <em>Téo Lemane</em>, Eric Guédon, and Jacques Nicolas </div> <div class="periodical"> <em>Microbial Genomics</em>, Sep 2021 </div> <div class="periodical"> Publisher: Microbiology Society, </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000654" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Link</a> <a href="/assets/pdf/papers/Siekaniec2021.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> <a href="https://github.com/gsiekaniec/ORI" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank"><i class="fa-brands fa-github"></i></a> </div> <div class="abstract hidden"> <p>This study aimed to provide efficient recognition of bacterial strains on personal computers from MinION (Nanopore) long read data. Thanks to the fall in sequencing costs, the identification of bacteria can now proceed by whole genome sequencing. MinION is a fast, but highly error-prone sequencing device and it is a challenge to successfully identify the strain content of unknown simple or complex microbial samples. It is heavily constrained by memory management and fast access to the read and genome fragments. Our strategy involves three steps: indexing of known genomic sequences for a given or several bacterial species; a request process to assign a read to a strain by matching it to the closest reference genomes; and a final step looking for a minimum set of strains that best explains the observed reads. We have applied our method, called ORI, on 77 strains of Streptococcus thermophilus . We worked on several genomic distances and obtained a detailed classification of the strains, together with a criterion that allows merging of what we termed ‘sibling’ strains, only separated by a few mutations. Overall, isolated strains can be safely recognized from MinION data. For mixtures of several non-sibling strains, results depend on strain abundance.</p> </div> </div> </div> </li></ol> </div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2024 Teo Lemane. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2781658a0a2b13ed609542042a859126"></script> <script defer src="/assets/js/common.js?e0514a05c5c95ac1a93a8dfd5249b92e"></script> <script defer src="/assets/js/copy_code.js?12775fdf7f95e901d7119054556e495f" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.min.js" integrity="sha256-rjmgmaB99riUNcdlrDtcAiwtLIojSxNyUFdl+Qh+rB4=" crossorigin="anonymous"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6" crossorigin="anonymous"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> <script src="/assets/js/vanilla-back-to-top.min.js?f40d453793ff4f64e238e420181a1d17"></script> <script>addBackToTop();</script> <script type="module" src="/assets/js/search/ninja-keys.min.js?601a2d3465e2a52bec38b600518d5f70"></script> <ninja-keys hidebreadcrumbs noautoloadmdicons placeholder="Type to start searching"></ninja-keys> <script>let searchTheme=determineComputedTheme();const ninjaKeys=document.querySelector("ninja-keys");"dark"===searchTheme?ninjaKeys.classList.add("dark"):ninjaKeys.classList.remove("dark");const openSearchModal=()=>{const e=$("#navbarNav");e.hasClass("show")&&e.collapse("hide"),ninjaKeys.open()};</script> <script>const ninja=document.querySelector("ninja-keys");ninja.data=[{id:"nav-about",title:"about",section:"Navigation",handler:()=>{window.location.href="/"}},{id:"nav-news",title:"news",description:"",section:"Navigation",handler:()=>{window.location.href="/news/"}},{id:"nav-publications",title:"publications",description:"",section:"Navigation",handler:()=>{window.location.href="/publications/"}},{id:"nav-repositories",title:"repositories",description:"",section:"Navigation",handler:()=>{window.location.href="/repositories/"}},{id:"nav-talks",title:"talks",description:"",section:"Navigation",handler:()=>{window.location.href="/talks/"}},{id:"nav-teaching",title:"teaching",description:"",section:"Navigation",handler:()=>{window.location.href="/teaching/"}},{id:"news-lt-code-gt-kmviz-lt-code-gt-first-release-is-available-lt-i-gt-v0-3-1-lt-i-gt",title:"&lt;code&gt;kmviz&lt;/code&gt; first release is available (&lt;i&gt;v0.3.1&lt;/i&gt;)",description:"",section:"News",handler:()=>{window.location.href="/news/kmviz-first-release/"}}];</script> <script src="/assets/js/shortcut-key.js"></script> </body> </html>